docs/content/040-adoption-guides/040-nexus-with-nextjs.md

```diff
@@ -0,0 +1,328 @@
+---
+title: Nexus With Next.js
```

Comment:
I think the title should match with the pattern of other adoption guides. Either rename this to `Next.js Users` or rename the others to be e.g. `Nexus with Prisma`, `...`. Though I don't know how that's going to work for the `Nexus Framework Users` title... So maybe `Next.js Users` is best & most consistent?